concepts/for-loops/.meta/config.json

```diff
@@ -1,9 +1,8 @@
 {
-  "blurb": "TODO: add blurb for for-loops concept",
+  "blurb": "For loops allow for iteration over a specified range.",
   "authors": [
     "samuelteixeiras",
     "ystromm"
   ],
-  "contributors": [
-  ]
-}
+  "contributors": []
+}
```

Comment:
Missing end of line for this file.  Generally, text files are expected to not have data after the last line.

Reply:
This was done automatically by the VSCode JSON formatter...

Reply:
Yeah, the VSCode editor makes this mistake I noticed last week while pairing with a student.  It is unfortunate.